src/components/Drum.js

```diff
@@ -0,0 +1,41 @@
+import DrumPad from "./DrumPad";
+import React, { useEffect } from 'react';
+
+function Drum(props) {
```

Comment:
You can use something like this directly:
```suggestion
function Drum({ kit }) {
```